src/components/CCPDashboard/ScanPatientButton.tsx

```diff
@@ -19,12 +20,16 @@ const useStyles = makeStyles({
 
 export const ScanPatientButton = () => {
```

Comment:
I am not familiar with best practices, but I would assume that a button that links to another page is quite common. Is it worth pulling out into common components instead?

Reply:
i tend to agree, but for now it's not super necessary - we can revisit this after all our flows are working correctly and we want to do cleanup/refactoring

Reply:
yup that'd probably be best - if we refactored all the buttons that redirect to another page, but yeah probably best for another time